walkthroughs/howto-outlier-detection/README.md

```diff
@@ -398,3 +398,96 @@ All resources created in this walkthrough can be deleted via:
 ```bash
 ./deploy.sh delete && ./mesh.sh down
 ```
+
+## Part 2: Using Virtual Gateway
```

Comment:
You should add a note above to say to keep the resources if they want to see an example using Virtual Gateways.